_posts/2019-02-10-circuit-simulator-post.markdown

```diff
@@ -17,18 +17,18 @@ Circuit Simulator really is a _simulator_ for electrical circuits, calling it a
 
 Sam and I picked it up in October of 2015. We had each worked on a few small games, and had planned ideas together, but this was our first time coding together. We decided on the most vanilla JS possible, with a goal of producing something over worrying about pushing our boundaries too much. You can see the origins of these blossoming, young professional devs in the [CircuitGame Repo](https://github.com/SylverStudios/CircuitGame).
 
-Solving circuit diagrams in school was one of the few things I liked about the electrical engineering classes I took, so I figured everyone felt that way :facepalm:. The first pass at circuit simulator was not only impossible, but it was completely devoid of fun. We called it a smashing success.
+Solving circuit diagrams in school was one of the few things I liked about the electrical engineering classes I took, so I figured everyone felt that way :face-palm:. The first pass at circuit simulator was not only impossible, but it was completely devoid of fun. We called it a smashing success.
```

Comment:
Was this an autocorrect change? If so, I'd guess it was a mistake, and will break the emoji 🚨

Reply:
shoot, was trying to joke about the slack one `:face_palm:` but that's an underscore. what do you suggest?